tests/test_feedbooks.py

```diff
@@ -43,12 +53,14 @@ def sample_file(self, filename):
         data = open(path).read()
         return data
 
-    def test_rights_for_entry(self):
+    def test_rights_uri_from_feedparser_entry(self):
         entry = dict(rights=LIFE_PLUS_70,
-                     source='gutenberg.net.au',
-                     publication_year="1922")
-        rights = self.importer.rights_for_entry(entry) 
-        eq_(RightsStatus.CC_BY_NC, rights)
+                     source='gutenberg.net.au')
+        expect = RehostingPolicy.rights_uri(
+            LIFE_PLUS_70, 'gutenberg.net.au', None
+        )
```

Comment:
Had to add a test where publication_year is None because that's what happens when we try to determine the rights URI based on incomplete feedparser data.